custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/internal_templates/icon_more_info_new.yaml

```diff
@@ -59,11 +59,15 @@ icon_more_info_new:
                 ulm_icon_double_tap_service: "[[[ return variables.ulm_icon_double_tap_service; ]]]"
                 ulm_icon_double_tap_service_data: "[[[ return variables.ulm_icon_double_tap_service_data]]]"
                 ulm_custom_popup: "[[[ return variables.ulm_custom_popup; ]]]"
+                ulm_notification_state: "[[[ return variables.ulm_notification_state; ]]]"
+                ulm_notification_icon: "[[[ return variables.ulm_notification_icon; ]]]"
+                ulm_notification_color: "[[[ return variables.ulm_notification_color; ]]]"
+                ulm_notification_bg_color: "[[[ return variables.ulm_notification_bg_color; ]]]"
               styles:
                 card:
                   - overflow: >
                       [[[
-                        if (entity.state == "unavailable"){
+                        if (entity.state == "unavailable" || entity.state == variables.ulm_notification_state){
```

Comment:
This allow us to only specify notification based on the current entity.

Reply:
Nothing wrong with that but it would be great to make it more "open", allowing notification to be based on any entity.